plugin.py

```diff
@@ -1081,8 +1081,9 @@ def run(self) -> None:
                     status = plugin.testitems.testitemstatus[filepath][idx]['status']
                     kind = TESTITEM_KINDS.get(status, sublime.KIND_AMBIGUOUS)
                     details = ", ".join(testitem.get('option_tags') or [])
+                    location = "{}:{}".format(filepath, testitem['range']['start']['line'])
```

Comment:
```suggestion
                    location = "{}:{}".format(filepath, testitem['range']['start']['line'] + 1)
```

Need to add 1 here because LSP positions are 0-based.